in_toto/model_test.go

```diff
@@ -595,10 +596,10 @@ func TestValidateLayout(t *testing.T) {
 				Type:    "layout",
 				Expires: "2020-02-27T18:03:43Z",
 				Keys: map[string]Key{
-					"deadbeef": Key{KeyId: "deadbeef"},
+					"deadbeef": Key{KeyId: "deabeef"},
```

Comment:
?

Reply:
mh this one is interesting... it will fail only with `deabeef` instead of `deadbeef`, because then the map key and value keyID match. No idea, what we actually want to test here.

Reply:
Fixed in https://github.com/in-toto/in-toto-golang/pull/56/commits/c25b937d9e9d022e5ecb8e82c6b56fa892dba6c7